README.md

```diff
@@ -45,29 +45,29 @@ Once the environment settings has been sourced, you are ready to proceed with th
 ### Quick start example
 
 ```bash
-python3 fpgaperf.py --toolchain vivado --project oneblink --board arty-a35t
+python3 fpgaperf.py toolchain=vivado project=oneblink board=arty-a35t
```

Comment:
Please don't change the syntax / format here.

Reply:
Unfortunately that's a side effect of using Hydra. Hydra takes control of the argparser of the app and doesn't allow user defined args. What are the consequences of changing the syntax?